PersonStageThree/src/com/beyond/person/core/service/interfaces/BasePersonAction.java

```diff
@@ -0,0 +1,27 @@
+package com.beyond.person.core.service.interfaces;
```

Comment:
@maria-salmasian interface should be in service package

Reply:
it is